src/App.js

```diff
@@ -29,41 +29,45 @@ class App extends Component {
       points: 0,
     };
 
+
     // Uncomment this to spawn a single test item
-    //const testItem = this.spawnItem(Date.now());
-    //this.state.items.push(testItem);
+    const testItem = this.spawnItem(Date.now());
+    this.state.items.push(testItem);
 
     // Uncomment this to automatically spawn new items
     this.enableSpawner();
-
-    console.log(this.state);
   }
 
-  onItemClicked = () => {
-    // Fill this in!
+  onItemClickedCallback = () => {
+    this.setState({
+      points: this.state.points + 1,
+    });
   }
 
+
   render() {
     const items = this.state.items.map((item, i) => {
-      return <GameItem
-               height={item.height}     // Height - used for a CSS style to position on the screen
-               layer={100 + i}          // Layer - used for a CSS style to show items on-top of bg
-               key={item.id}            // Key - to help React with performance
 
-               // Additional props (event callbacks, etc.) can be passed here
-             />;
+      console.log(item)
+      return <GameItem
+        height={item.height}     // Height - used for a CSS style to position on the screen
+        layer={100 + i}          // Layer - used for a CSS style to show items on-top of bg
+        key={item.id}            // Key - to help React with performance
+        type={item.type}
+        points={this.onItemClickedCallback}
```

Comment:
This name is unclear. Is Points a variable? A callback? Make it clear what kind of data this is from the name.